.buildscript/android-ui-tests.gradle

```diff
@@ -11,7 +11,8 @@ android {
 
 dependencies {
   androidTestImplementation project(":workflow-ui:internal-testing-android")
-  androidTestImplementation Deps.get("test.androidx.espresso.core")
-  androidTestImplementation Deps.get("test.androidx.junitExt")
-  androidTestImplementation Deps.get("test.leakcanaryInstrumentation")
+
+  implementation(libs.androidx.test.espresso.core)
```

Comment:
Why the change to implementation from androidTestImplementation?

Reply:
I believe this was a copy/paste error, which then got caught by Lint. This explains the resolution errors.  I hadn't had a chance to actually switch to this branch and check it out yet, so thanks for saving me the hassle of debugging. :D